tests/Rule/RuleTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabberworm\CSS\Tests\Value;
+
+use PHPUnit\Framework\TestCase;
+use Sabberworm\CSS\Parsing\ParserState;
+use Sabberworm\CSS\Settings;
+use Sabberworm\CSS\Rule\Rule;
+use Sabberworm\CSS\Value\RuleValueList;
+use Sabberworm\CSS\Value\ValueList;
+
+/**
+ * @covers \Sabberworm\CSS\Rule\Rule
+ */
+final class RuleTest extends TestCase
+{
+    /**
+     * @return array<string, array{0: string, 1: array<int, string>}>
```

Comment:
```suggestion
     * @return array<string, array{0: string, 1: list<class-string>}>
```

See:
https://phpstan.org/writing-php-code/phpdoc-types#lists
https://phpstan.org/writing-php-code/phpdoc-types#class-string

Reply:
Thanks.  I didn't know about `class-string`, and had wrongly assumed `list` was a more general type than `array`.  Changed as suggested.